lib/controller/ProfielController.class.php

```diff
@@ -202,7 +202,7 @@ public function stamboom($uid = null) {
 
 	public function verjaardagen() {
 		require_once 'view/VerjaardagenView.class.php';
-		$body = new VerjaardagenView('alleverjaardagen');
+		$body = new AlleVerjaardagenView(null);

```

Comment:
Zou het niet al voldoende zijn om in de `SmartyTemplateView` dit neer te zetten:
```
public function __construct($model = null, $titel = false) {
```